packages/edit-site/src/utils/normalize-post-id-for-post-type.js

```diff
@@ -0,0 +1,9 @@
+const POST_TYPES_THAT_USE_STRING_BASED_IDS = [
+	'wp_template',
+	'wp_template_part',
+];
```

Comment:
Perhaps this information could come from entities config? It feels like we should have one single source of truth for this information rather than have it stuff away in a `const` within a utility.

Reply:
This is interesting. Could use use `Number.isNaN(Number(postId))` on any ID and if it's a number we convert if not we dont? So that we don't need to have this array of types that use string ids?

cc @jsnajdr @youknowriad

Reply:
I updated this to use the utility from https://github.com/WordPress/gutenberg/pull/52120.